tools/dds/dds-sniffer/rs-dds-sniffer.cpp

```diff
@@ -197,14 +245,12 @@ bool dds_sniffer::init( librealsense::dds::dds_domain_id domain,
 
     if( ! _print_machine_readable )
     {
+        std::cout << "rs-dds-sniffer (" << print( _participant.guid(), false ) << ") ";
         if( snapshot )
-        {
-            std::cout << "rs-dds-sniffer taking a snapshot of domain " << domain << std::endl;
-        }
+            std::cout << "taking a snapshot of ";
         else
-        {
-            std::cout << "rs-dds-sniffer listening on domain " << domain << std::endl;
-        }
+            std::cout << "listening on ";
```

Comment:
It is better to use curly brackets even on one line if, I have seen bugs that could be avoided that way.

Reply:
Sure, but for simple statements not worth the 4 extra lines.